crates/g3-core/src/lib.rs

```diff
@@ -962,6 +975,30 @@ impl<W: UiWriter> Agent<W> {
                     16384 // Conservative default for other Databricks models
                 }
             }
+            "ollama" => {
+                // Ollama model context windows based on model name
+                if model_name.contains("qwen3-coder") {
```

Comment:
did g3 invent these, or are they real? I'm asking b/c I keep getting errors when I'm using Anthropic, so just curious about the impact here.